src/js/media-time-range.js

```diff
@@ -443,58 +461,45 @@ class MediaTimeRange extends MediaChromeRange {
     setBooleanAttr(this, MediaUIAttributes.MEDIA_ENDED, value);
   }
 
-  getRelativeValues() {
-    const defaultRelativeValues = super.getRelativeValues();
-    if (!this.mediaEnded) return defaultRelativeValues;
-    return {
-      ...defaultRelativeValues,
-      relativeValue: defaultRelativeValues.relativeMax,
-    };
-  }
-
   /* Add a buffered progress bar */
-  getBarColors() {
-    let colorsArray = super.getBarColors();
-    const { range } = this;
-    const relativeMax = range.max - range.min;
-    const buffered = this.mediaBuffered;
+  updateBar() {
+    super.updateBar();
 
-    if (!buffered.length || !Number.isFinite(relativeMax) || relativeMax <= 0) {
-      return colorsArray;
+    const buffered = this.mediaBuffered;
+    if (!buffered.length) {
+      return;
     }
 
     // Find the buffered range that "contains" the current time and get its end.
-    // If none, just assume the start of the media timeline/range.min for
+    // If none, just assume the start of the media timeline for
     // visualization purposes.
     let relativeBufferedEnd;
+
     if (!this.mediaEnded) {
       const currentTime = this.mediaCurrentTime;
-      const [, bufferedEnd = range.min] = buffered.find(
+      const [, bufferedEnd = 0] = buffered.find(
```

Comment:
question(blocking): Shouldn't this be `bufferedEnd = this.mediaSeekableStart]`?

Reply:
I think you're right. will update!